data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,122 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				goto next
+			label poignant
+			action
+				set haiinsight
+			`	There is a quiet shuffling as some of the elders sit up a little straighter. The Elder says, "A common ground between rivals, a fearsome thing indeed."`
+				goto next
+			label constructive
+			action
+				set haiinsight
+			`	A roomful of laughter engulfs your words, but many Elders appear offended by that mockery. The Elder who questioned you explains. "Elders and Unfettered have different views of what constitutes a 'constructive discussion.' You lack the biases of a Hai, so if you say it was constructive, I believe you." The tone of the room changes after that, as the laughter ends. Most Elders appear to agree with your assessment, and many look ashamed.`
+			label next
+			`	Sayari gives you a meaningful look and whispers, "A lot of them paid attention to that..."`
+			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are all of you cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`
+			`	As the crescendo of her impassioned statement crests to its conclusion there is a growing rumble of ascent and the room is suddenly filled with the angry cries of those both emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen elders jump to their feet but the cacophony is not so easily restrained. At your side Sayari's expression is ashen but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`
+			`	There is a shift in the sound of the room and your attention is drawn to one Hai in particular, a particularly elderly man who struggles to his feet in a series of pained stages. The effect on the assembly is profound though. A ripple of silence flows outward from his position and elders all around promptly drop back into their seats as it reaches them, leaving a scant handful who were too slow to react to have their final words trail off awkwardly. Only the last speaker remains on her feet, the moderately weakly announcing what you know to be, "The council hears Aimet."`
```

Comment:
```suggestion
			`	There is a sudden shift in the sound of the room. Your attention is drawn to one Hai in particular, a particularly elderly man who struggles to his feet in a series of pained stages. The effect on the assembly is profound, though: a ripple of silence flows outward from his position and Elders all around promptly drop back into their seats as it reaches them, leaving a scant handful, who were too slow to react, to have their final words trail off awkwardly. Only the last speaker remains on her feet, while the moderator weakly announces, "The Council hears Aimet."`
```
Interesting to see. This suggestion can be changed, I expect people to ask about it.

Reply:
I'm not a fan of how this one changes up the pacing, and I think the use of more advanced punctuation detracts from the reader's focus on the event.
Also the moderator is speaking Hai, you've just heard the phrase enough now that you know what it means.